app/helpers/users_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```

Comment:
If we're not using this helper why do we commit this helper?
Please remove this